src/main/java/task/Storage.java

```diff
@@ -0,0 +1,98 @@
+package task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+
+    private final static String FILE_PATH = "data/duke.txt";
+
+    /**
+     * Reads data from data file.
+     * 
```

Comment:
Java docs are not meant to have the space between each line. Coding style violation